src/main/groovy/com/hcl/security/appscan/gradle/tasks/SecurityTask.groovy

```diff
@@ -35,8 +39,9 @@ abstract class SecurityTask extends DefaultTask {
         }
     }
 
-    public static void clearTargets() {
+    public static void actionsWhenBuildFinished() {
```

Comment:
Can you change the function name from "actionsWhenBuildFinished" to "reset" or "cleanUp" to shorten the name and make it more clear what's happening?